commons-api/src/main/java/com/connexta/transformation/commons/api/status/Transformation.java

```diff
@@ -57,9 +66,12 @@
   /**
    * Returns a {@link MetadataTransformation}, for the given type.
    *
+   * @param metadataType the type of metadata to retrieve
    * @return A {@link MetadataTransformation} or a empty if it doesn't have one of that type.
+   * @throws TransformationNotFoundException if no metadata of the given type is being generated
+   *     from this transformation
    */
-  Optional<MetadataTransformation> getMetadata(String metadataType);
```

Comment:
Why is this not optional, compared to before? Also may need to update the return in the javadoc

Reply:
Must of been in a different state when I commented this... but I am talking about this https://github.com/connexta/ion-transformation-storage/pull/4/files#diff-a23a1e981cc4634d75540e210a6169f1R74

Reply:
I did it to be consistent with the other get methods on the manager where they throw the not found exception as opposed to returning optionals